libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -468,7 +470,12 @@ private void initializeLocationLayer(MapView mapView, MapboxMap map) {
     Context context = mapView.getContext();
     int locationLayerStyleRes = ThemeSwitcher.retrieveNavigationViewStyle(context,
       R.attr.navigationViewLocationLayerStyle);
-    locationLayer = new LocationLayerPlugin(mapView, map, null, locationLayerStyleRes);
+
+    LocationLayerOptions locationLayerOptions =
+      LocationLayerOptions.createFromAttributes(context, locationLayerStyleRes);
+    locationLayerOptions = locationLayerOptions.toBuilder().minZoom(NAVIGATION_MINIMUM_MAP_ZOOM).build();
```

Comment:
@LukasPaczos can we extract this into the `styles.xml` so it aligns with https://github.com/mapbox/mapbox-navigation-android/pull/1347 once that lands as well?